cli/README.md

```diff
@@ -57,6 +57,95 @@ yo mern:component Sidebar --stateless
 
 Produces `app/components/Sidebar.js`.
 
```

Comment:
Done in https://github.com/MakingSense/mern-seed/pull/16/commits/8758aa9d74fc792b1d03f23a853b20c7d7c08db0.